spec/lib/danger/ci_sources/ci_source_spec.rb

```diff
@@ -5,35 +5,36 @@
     it "returns list of CI subclasses" do
       expect(described_class.available_ci_sources.map(&:to_s)).to match_array(
         [
-          "Danger::LocalGitRepo",
```

Comment:
I've added `Danger::XcodeCloud` as a source here, and then sorted them alphabetically for better organization.